lib/generators/react_on_rails/react_generator.rb

```diff
@@ -0,0 +1,132 @@
+require "rails/generators"
+require File.expand_path("../generator_helper", __FILE__)
+include GeneratorHelper
+
+module ReactOnRails
+  module Generators
+    class ReactGenerator < Rails::Generators::Base
+      hide!
+      source_root File.expand_path("../templates", __FILE__)
+
+      # --with-redux
+      class_option :with_redux,
+                   type: :boolean,
+                   default: false,
+                   desc: "Include Redux package"
+      # --with-hello-world-example
+      class_option :with_hello_world_example,
+                   type: :boolean,
+                   default: false,
+                   desc: "Include a simple Hello World Example. Note that this example's implementation varies " \
+                         "depending on the other options chosen."
+      # --with-server-rendering
+      class_option :with_server_rendering,
+                   type: :boolean,
+                   default: false,
+                   description: "Enable server-rendering"
+
+      def symlink_client_assets_to_app_assets
+        empty_directory("client/app/assets")
+        empty_directory_with_keep_file("app/assets/fonts") unless dest_dir_exists?("app/assets/fonts")
+        empty_directory_with_keep_file("app/assets/images") unless dest_dir_exists?("app/assets/images")
+        symlink_dest_file_to_dest_file("app/assets/fonts", "client/app/assets/fonts")
+        symlink_dest_file_to_dest_file("app/assets/images", "client/app/assets/images")
+      end
+
+      def update_git_ignore
+        data = "# React on Rails\n"
+        data << "npm-debug.log\n"
+        data << "node_modules\n"
+        data << "\n"
+        data << "# Generated js bundles\n"
+        data << "/app/assets/javascripts/generated/*\n"
+
+        if dest_file_exists?(".gitignore")
+          append_to_file(".gitignore", data)
+        else
+          puts_setup_file_error(".gitignore", data)
+        end
+      end
+
+      def update_application_js
+        data = "\n"
```

Comment:
use 

```ruby
data = <<-DATA.strip_heredoc
  // It is important that generated/vendor-bundle must be before bootstrap since it is exposing jQuery and jQuery-ujs
  //= require generated/vendor-bundle
etc.
DATA
```